easybuild/easyconfigs/a/abtem/abtem-1.0.0b24-foss-2020b.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonPackage'
+
+name = 'abtem'
+version = '1.0.0b24'
+
+homepage = 'https://imageio.github.io'
+description = """abTEM provides a Python API for running simulations
+of Transmission Electron Microscopy images. It is written entirely in
```

Comment:
@schiotz Maybe this should be trimmed down to just the first sentence?

Reply:
Fixed.